### PR TITLE
support --run flag for `apono access use`

### PR DESCRIPTION
**Pull request by naorgu**

### CODE REVIEW

File: pkg/access/commands/use.go [hunk 1]
```diff
@@ -1,60 +1,68 @@
...
+			}
+
+			if run {
+				return executeAccessDetails(cmd.Context(), accessID, accessDetails)
+			}
+
+			err = verifyOutputFormatIsSupported(cmd.Context(), client, accessID, outputFormat)
 			if err != nil {
 				return err
 			}
```
Comment (DmitriyO) [inline pkg/access/commands/use.go:50]: don't forget to move up

File: pkg/access/commands/use.go [hunk 2]
```diff
@@ -1,60 +1,68 @@
...
 
 			var output string
 			switch outputFormat {
 			case "cli":
```
Comment (DmitriyO) [inline pkg/access/commands/use.go:54]: I'm missing something, where is the "no output format" option (default)?

File: pkg/access/commands/use.go [hunk 3]
```diff
@@ -78,6 +82,35 @@ func AccessDetails() *cobra.Command {
...
+			return nil
+		}
+	}
+
+	return fmt.Errorf("unsupported output format: %s. use one of: %s", outputFormat, strings.Join(session.ConnectionMethods, ", "))
+}
+
+func executeAccessDetails(ctx context.Context, accessID string, accessDetails *clientapi.AccessSessionDetailsClientModel) error {
+	if accessDetails.GetCli() == "" {
+		return fmt.Errorf("--run flag is not supported for access id %s", accessID)
```
Comment (DmitriyO) [inline pkg/access/commands/use.go:107]: -> flag is not supported for session id

Comment (DmitriyO) [reply pkg/access/commands/use.go:107]: in general I would go over the error messages with Ofir/Sharon

File: pkg/access/commands/use.go [hunk 4]
```diff
@@ -78,6 +82,35 @@ func AccessDetails() *cobra.Command {
...
+	}
+
+	err := exec.CommandContext(ctx, "sh", "-c", accessDetails.GetCli()).Run() //nolint:gosec // This is a command that should be executed for the user
+	if err != nil {
+		return fmt.Errorf("error executing command:\n%s\n%w", accessDetails.GetCli(), err)
+	}
```
Comment (DmitriyO) [inline pkg/access/commands/use.go:113]: does this execute the command in the current shell?
I must be using the same shell the cli is run from to inherit all the variables, params and credentials and set everything properly for the user

**Review by DmitriyO — APPROVED** [review]
Few comments to verify verore merge

File: pkg/access/commands/use.go (hunk 2 above)
Comment (naorgu) [reply pkg/access/commands/use.go]: default is cli
```go
flags.StringVarP(&outputFormat, outputFlagName, "o", "cli", "output format")
```

**Pull request merged**